includes/class-cli.php

```diff
@@ -158,4 +204,37 @@ private function return_response( $response = array() ) {
 			\WP_CLI::error( 'There was a porblem. All of the content didn\'t sync.' );
 		}
 	}
+
+	/**
+	 * Parse the associative arguments with the Press Sync settings.
+	 *
+	 * @since NEXT
+	 * @param  array $assoc_args The associative args from the command line.
+	 * @return array
+	 */
+	private function parse_assoc_args( $assoc_args ) {
+		$valid_keys    = array_keys( $this->plugin->parse_sync_settings() );
+		$valid_keys[]  = 'verbose';
+		$prepared_args = [];
+
```

Comment:
So, I get from the title that we're "parsing the associative arguments", but what does that actually mean to a developer who might look at this code later? Can we provide an example input and an example output in the doc block so it's more clear what kinds of things are being stripped out of the args? Without named functions processing these foreach loops, we're just kind of left knowing that certain kinds of args are getting excluded.

Reply:
I added some elaboration to the docblock for clarification.